client/layout/masterbar/logged-in.jsx

```diff
@@ -378,6 +344,151 @@ class MasterbarLoggedIn extends Component {
 		);
 	}
 
+	renderSiteMenu() {
+		const { sectionGroup, currentSelectedSiteSlug, translate, siteTitle, siteUrl } = this.props;
+
+		// Only display on site-specific pages.
+		if ( sectionGroup !== 'sites' || ! currentSelectedSiteSlug ) {
+			return null;
+		}
+
+		return (
+			<>
+				<Item
+					className="masterbar__item-my-site"
+					url={ siteUrl }
+					icon={ <span className="dashicons-before dashicons-admin-home" /> }
+					tipTarget="Menu"
+					subItems={ [ { label: translate( 'Visit your site' ), url: siteUrl } ] }
```

Comment:
I believe this should be `Visit Site` as this is what Core uses.

```suggestion
					subItems={ [ { label: translate( 'Visit Site' ), url: siteUrl } ] }
```